cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage.hs

```diff
@@ -56,6 +56,10 @@ unitTests iom knownMigrations =
         [ test "simple tx" BabTx.addSimpleTx
         , test "simple tx in Shelley era" BabTx.addSimpleTxShelley
         , test "consume utxo same block" BabTx.consumeSameBlock
+        , test "migrate to consume tx-out" BabTx.consumeTxOutMigrated
```

Comment:
Maybe it's worth adding these tests in a new testGroup dedicated to the flags? This group is for simple transactions without a rollback and there is a separate group for rollbacks. In that case though they would be scattered